cmd/up/activate.go

```diff
@@ -63,6 +63,10 @@ func (up *upContext) activate() error {
 		return err
 	}
 
+	if v, ok := app.ObjectMeta().Annotations[model.OktetoAutoCreateAnnotation]; !ok || v != model.OktetoUpCmd {
+		return fmt.Errorf("resource '%s' already exists. Please consider changing the name of autocreate", up.Dev.Name)
```

Comment:
I thin it would be better to leave the error as resource %s doesn't exist, and then add an `info` like: "use a different name in your okteto.yaml, or remove the autocreate property"